packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -205,6 +261,25 @@ const Select = <
     [onChangeProp]
   );
 
+  const handleDelete = useCallback(
+    (itemToDelete: string) => {
+      if (!Array.isArray(internalSelectedValues)) {
+        return;
+      }
+
+      const newValue = internalSelectedValues!.filter(
+        (item: string) => item !== itemToDelete
+      );
+
+      const syntheticEvent = {
+        target: { value: newValue },
+      } as SelectChangeEvent<Value>;
+
+      onChange(syntheticEvent, null);
+    },
+    [internalSelectedValues, onChange]
+  );
```

Comment:
What does `handleDelete` do? Can we name it something like `deleteItem` or `deleteOption`? Maybe even `removeSelection`? At a glance, I'm not sure what this is doing.